dspace-api/src/main/java/org/dspace/app/bulkedit/DSpaceCSVLine.java

```diff
@@ -205,6 +205,13 @@ protected String valueToCSV(List<String> values, String valueSeparator) {
 
             s = str.toString();
         }
+        // Sanitise for Excel
+        s = s.replaceAll("^[\\r\\n]+", "");
+        if (s.length() > 1 && "+@=-".indexOf(s.charAt(0)) != -1) {
+            s = s.substring(1);
```

Comment:
Please correct me if I'm wrong, but this appears to be **removing** the special characters `+`, `+`, `@`, `=` or `-` if they appear at the beginning of a metadata value.

I'm concerned there's no way to "unsanitize" this data on re-import of the CSV.  While I'm not sure if many metadata values would **start with** those special characters, it seems wrong to remove them without a way to restore them on import.